index.js

```diff
@@ -28,15 +31,12 @@ app.use((err, req, res, next) => {
     const status = err.status || 500
 
     // respond to client
-    res.status(status).json({
-        error: {
-            message: err.message
-        }
-    })
+    response(res, 'fail', "", err.message, status)
 
     // respond to dev
     console.log(err.message)
 })
 
 // running app
-app.listen(3000 || process.env.PORT, () => console.log('listen on port ' + 3000 || process.env.PORT))
\ No newline at end of file
+port = 3000 || procces.env.PORT
```

Comment:
```suggestion
const port = procces.env.PORT || 3000 
```

fallback itu ditaroh diakhir bi , kalo di depan nanti misal process.env.PORT nya di set dia gak akan dipake